src/CoinbaseWalletSDK.ts

```diff
@@ -203,6 +204,16 @@ export class CoinbaseWalletSDK {
     return this._relay?.getQRCodeUrl() ?? null;
   }
 
+  /**
+   * Official Coinbase Wallet logo for developers to use on their frontend
+   * @param type Type of wallet logo (1-6), please refer to Coinbase Wallet SDK Docs for details
+   * @param width Width of the logo to be rendered
+   * @returns SVG Data URI
+   */
+  public getCoinbaseWalletLogo(type: number, width = 240): string {
```

Comment:
Could we use a string here instead with only a few allowed inputs?

something like `type logoType = "circle" | "square" | "text" | "text-with-logo";`

Would make it more readable and prevent logos suddenly changing in case we reorder the numbering.

Reply:
Agree with @brianchitester & @NikitaVr 
Preference for "friendlier" text label for type like shared above